02-ml-normalization.mm1

```diff
@@ -39,6 +39,15 @@ theorem eSubstitution_in_same_exists {x: EVar} (psi phi: Pattern x):
 theorem sSubstitution_in_same_mu {X: SVar} (psi phi: Pattern X):
   $ Norm (s[ psi / X ] mu X phi) (mu X phi) $ =
   '(sSubstitution_fresh sFresh_mu_same_var);
+theorem appCtxL_strict {box: SVar} (phi1 ctx: Pattern box) (phi2: Pattern):
+  $ Norm (app[ phi1 / box ] (app ctx phi2)) (app (app[ phi1 / box ] ctx) phi2) $ =
+  '(appCtxL sFresh_disjoint);
+theorem appCtxR_strict {box: SVar} (phi2 ctx: Pattern box) (phi1: Pattern):
+  $ Norm (app[ phi2 / box ] (app phi1 ctx)) (app phi1 (app[ phi2 / box ] ctx)) $ =
+  '(appCtxR sFresh_disjoint);
+theorem appCtxNested_strict {box1 box2: SVar} (ctx2 phi: Pattern box1 box2) (ctx1: Pattern box1):
```

Comment:
Why are these ones called `_strict` instead of `_disjoint`?

Reply:
We have used the suffix strict before for stuff like `exists_generalisation_strict` and such, we could just rename all of them to `disjoint` though, I don't mind.